lib/utils/url_launcher.dart

```diff
@@ -209,7 +210,7 @@ class UrlLauncher {
 
   void openUrlInAppBrowser() async {
     if (url != null) {
-      if (Platform.isAndroid) {
+      if (!kIsWeb && Platform.isAndroid) {
```

Comment:
Pls use `PlatformInfos.isAndroid`

<img width="534" alt="Screenshot 2023-08-31 at 09 20 44" src="https://github.com/linagora/twake-on-matrix/assets/99852347/702722d6-32a8-48e2-94ab-82cec7f9daf9">

Reply:
Thank you I didn't see it